src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -298,6 +298,40 @@ object SampleWdl {
     override lazy val rawInputs = Map(ThreeStep.PatternKey -> "." * 10000)
   }
 
+  object WorkflowOutputsWithFiles extends SampleWdl {
+    override def wdlSource(runtime: String = "") =
+      """
+        task A {
+          command {
+            echo "Enfin un peu de francais pour contrer ce raz-de-marée anglais !" > out
+            echo "Jacques Chirac fait du jetski sur la Seine en costume traditionnel russe" > out2
```

Comment:
We have set a pretty [high bar](http://www.theguardian.com/world/video/2014/jun/13/george-bush-sr-celebrates-90th-with-skydive-video) here I suppose

:us: